draft-ietf-privacypass-architecture.md

```diff
@@ -199,6 +199,42 @@ does not reply to the Origin's challenge with a new request.
 ~~~
 {: #fig-overview title="Privacy pass redemption and issuance protocol interaction"}
 
+The end-to-end flow for Privacy Pass involves three different types of contexts:
+
+- Redemption context: The interactions and set of information shared
+between the Client and Origin.
+- Issuance context: The interactions and set of information shared
+between the Client, Attester, and Issuer.
+- Attestation context: The interactions and set of information shared between
+the Client and Attester only, for the purposes of attesting the vailidity of the Client.
```

Comment:
Hm, this seems to be duplicated from the terminology above. Do we want to remove them there?

Reply:
I considered it, yeah.